### ISSUE TITLE
setup_args in top level setup.py created multiple times

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5146]
|[<img alt="zseil's avatar" src="https://avatars.githubusercontent.com/u/421894?s=50" width="50" height="50">](https://github.com/zseil)| @zseil reported|
|-|-|
|Trac ID|trac#5146|
|Type|defect|
|Created|2011-06-08 09:30:12Z|
|Branch|https://github.com/twisted/twisted/tree/setup-args-5146|

The `setup_args` dictionary in Twisted's top level `setup.py` is created multiple times, once for each subproject, due to wrong indentation level.

Attachments:

* [setup.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0e1/0e19644e35a2b71ff35115c2d7a68a65fd76ce7e/268859299e3e3b7e2393a1e73d202c6d4976372b.patch) (2593 bytes) - added by zseil on 2011-06-08 09:31:53Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5146 5146
type__defect defect
reporter__zseil zseil
priority__normal normal
milestone__ 
branch__branches_setup_args_5146 branches/setup-args-5146
branch_author__zseil zseil
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1307525412000000 1307525412000000
changetime__1309825429000000 1309825429000000
version__None None
owner__jesstess jesstess
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [32186]) Branching to 'setup-args-5146'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [32187]) apply setup.patch, add news file. refs [#5146](https://github.com/twisted/twisted/issues/5146)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/builders/lucid64-py2.6-select/builds/1027).

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess set status to `closed`|
|-|-|

(In [32223]) Merge setup-args-5146

Author: zseil
Reviewer: jesstess
Fixes: [#5146](https://github.com/twisted/twisted/issues/5146)

Fix the indentation level of the setup_args dictionary in Twisted's
top level setup.py so it isn't erroneously created several times.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess commented|
|-|-|

Thanks for the patch, zseil, and sorry for the delay in getting it reviewed!
